### PR TITLE
Add sever and client keys to eventbus

### LINKED ISSUE #7856: Grid: secure 0mq
The default 0mq event bus should be able to be secured via configuration.

### PR DESCRIPTION
**Pull request by raju249**
**Thanks for contributing to Selenium!**
**A PR well described will help maintainers to quickly review and merge it**

Before submitting your PR, please check our [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) guidelines.
Avoid large PRs, help reviewers by making them as simple and short as possible.

### Description

Fixes https://github.com/SeleniumHQ/selenium/issues/7856

### Motivation and Context

We had an unsecured implementation of zeromq, this PR adds security by using the CurveMQ server and client keys and certificates.

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Checklist

- [ ] I have read the [contributing](https://github.com/SeleniumHQ/selenium/blob/trunk/CONTRIBUTING.md) document.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java [hunk 1]
```diff
@@ -24,7 +24,9 @@
 import org.zeromq.SocketType;
 import org.zeromq.ZContext;
 import org.zeromq.ZMQ;
+import zmq.io.mechanism.curve.Curve;
 
+import javax.swing.plaf.ColorUIResource;
```
Comment (shs96c) [inline java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java]: Looks like a mistaken import. Please remove.

**Review by shs96c — COMMENTED** [review]
Question: I assume that it won't be necessary to share keys between event bus instances on different machines?

File: java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java (hunk 1 above)
Comment (raju249) [reply java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java]: Thanks @shs96c !

Removed the unused import.

For the question: I assumed the same. @adamgoucher can shed some light?

Comment (adamgoucher) [reply java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java]: I don't think so. the 0mq bus is something we control 100% and this is just a secure default. (I expect most large implementations will use a different bus anyways)

Comment (raju249) [reply java/server/src/org/openqa/selenium/events/zeromq/BoundZmqEventBus.java]: Thanks @adamgoucher

**Pull request merged**